Source/WebCore/css/CSSProperties.json

```diff
@@ -9163,5 +9024,146 @@
             "longname": "SVG",
             "url": "https://www.w3.org/TR/SVG11/"
         }
+    },
+    "shared-grammar-rules": {
+        "<alpha-value>": {
+            "grammar": ["<number>", "<percentage>"],
+            "specification": {
+                "category": "css-color",
+                "url": "https://www.w3.org/TR/css-color-4/#typedef-alpha-value"
+            }
+        },
+        "<marker-ref>": {
+            "grammar": ["<url>"],
+            "specification": {
+                "category": "svg",
+                "url": "https://www.w3.org/TR/SVG/painting.html#DataTypeMarkerRef"
+            }
+        },
+        "<absolute-size>": {
+            "grammar": ["xx-small", "x-small", "small", "medium", "large", "x-large", "xx-large", "xxx-large"],
+            "specification": {
+                "category": "css-fonts-4",
+                "url": "https://drafts.csswg.org/css-fonts-4/#valdef-font-size-absolute-size"
+            }
+        },
+        "<-webkit-absolute-size>": {
+            "grammar": [
+                {
+                    "value": "-webkit-xxx-large",
+                    "aliased-to": "xxx-large"
+                }
+            ],
+            "status": "non-standard"
+        },
+        "<relative-size>": {
+            "grammar": ["larger", "smaller"],
+            "specification": {
+                "category": "css-fonts-4",
+                "url": "https://drafts.csswg.org/css-fonts-4/#valdef-font-size-relative-size"
+            }
+        },
+        "<-webkit-relative-size>": {
+            "grammar": ["-webkit-ruby-text"],
+            "status": "non-standard"
+        },
+        "<gap-gutter>": {
+            "grammar": ["normal", "<length-percentage [0,inf]>"],
+            "specification": {
+                "category": "css-align",
+                "url": "https://drafts.csswg.org/css-align/#column-row-gap"
+            }
+        },
+        "<width-or-height-keyword>": {
+            "grammar": [
+                "intrinsic",
+                "min-intrinsic",
+                "min-content",
+                "webkit-min-content",
+                "max-content",
+                "webkit-max-content",
+                "webkit-fill-available",
+                "fit-content",
+                "webkit-fit-content"
```

Comment:
@weinig I think the webkit ones are missing the `-`? I guess it doesn't matter when the code is generated since it will generate the same `CSSValueWebkitX`, but it would be nice to follow the CSS values.

Reply:
Good catch! Thanks. Will fix it in the next pr.

Reply:
PR -> https://github.com/WebKit/WebKit/pull/6826